src/main/java/marquez/client/MarquezHttp.java

```diff
@@ -69,6 +82,10 @@ String post(URL url, @Nullable String json) {
         request.setEntity(new StringEntity(json, APPLICATION_JSON));
       }
 
+      if (apiKey != null) {
+        Utils.addAuthTo(request, apiKey);
+      }
+
```

Comment:
It looks like this could be added to a private method in this class to avoid duplication.
something like _addAuth(request)_